proxy/proxy.go

```diff
@@ -120,8 +121,8 @@ type Params struct {
 	// and the response messages during web socket upgrades.
 	ExperimentalUpgradeAudit bool
 
-	// When set, no access log is printed.
-	AccessLogDisabled bool
```

Comment:
This is a breaking change and I am sure we can have a better solution if we change it like this which might support also https://github.com/zalando/skipper/issues/867

Reply:
I wouldn't consider this a breaking change since it's a change of what is essentially an internal package in skipper. I understand that packages are public in Go, but I would not expect anyone to use the proxy package directly. Or do you think there is a use case for using the proxy package directly?

Reply:
I will close the PR as discussed with @njuettner . I doubt, that the idea is more a hack than needed. If we would have a default filter applied in routing package it would make more sense and create more value.